src/System.Drawing.Common/src/Configurations.props

```diff
@@ -4,6 +4,7 @@
     <BuildConfigurations>
       netcoreapp2.0-Windows_NT;
       netcoreapp2.0-Unix;
```

Comment:
As we discussed it may make since to have a RID-less implementation (potentially one that throws) to handle the *new RID* case, but not something you need to do in this commit.